### ISSUE TITLE
Sigh 1.5.0 has different verbose output compared to 1.4.0

### ISSUE THREAD

**Issue by iphil** [fastlane/fastlane#3835]
The useful information like identifer etc. is now missing in 1.5.0 

1.4.0
```
Specified provisioning profile: 'AppStore_North_America.mobileprovision'
Specified provisioning profile: 'AppStore__North_America_Share_Ext.mobileprovision'
Resigning nested application: '_floatsignTemp/Payload/NA.app/PlugIns/LocationShareExtension.appex'
Current bundle identifier is: 'Blub.na.shareExtension'
New bundle identifier will be: 'Blub.na.shareExtension'
Validating the new provisioning profile: AppStore_North_America_Share_Ext.mobileprovision
Profile app identifier prefix is 'XC'
Profile team identifier is ''
Resigning embedded frameworks using certificate: 'iPhone Distribution:, LLC'
Extracting entitlements from provisioning profile
Resigning application using certificate: 'iPhone Distribution: LLC (XC36KHSA2U)'
and entitlements from provisioning profile: AppStore_North_America_Share_Ext.mobileprovision
Current bundle identifier is: '.na'
New bundle identifier will be: '.na'
Validating the new provisioning profile: AppStore_North_America.mobileprovision
Profile app identifier prefix is 'XC36KHSA2U'
Profile team identifier is 'XC36KHSA2U'
Resigning embedded frameworks using certificate: 'iPhone Distribution: NORTH AMERICA, LLC (XC36KHSA2U)'
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftAVFoundation.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftContacts.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCore.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreAudio.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreBluetooth.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreData.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreGraphics.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreImage.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreLocation.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreMedia.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftDarwin.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftDispatch.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftEventKit.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftFoundation.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftLocalAuthentication.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftObjectiveC.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftUIKit.dylib: replacing existing signature
Extracting entitlements from provisioning profile
Resigning application using certificate: 'iPhone Distribution: NORTH AMERICA, LLC (XC36KHSA2U)'
and entitlements from provisioning profile: AppStore__North_America.mobileprovision
Repackaging as blubNorthAmericaWithSwift.ipa
Process complete`
```

1.5.0

```
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftAVFoundation.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftContacts.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCore.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreAudio.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreBluetooth.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreData.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreGraphics.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreImage.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreLocation.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftCoreMedia.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftDarwin.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftDispatch.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftEventKit.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftFoundation.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftLocalAuthentication.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftObjectiveC.dylib: replacing existing signature
_floatsignTemp/Payload/blubNA.app/Frameworks/libswiftUIKit.dylib: replacing existing signature
Extracting entitlements from provisioning profile
Resigning application using certificate: 'iPhone Distribution: NORTH AMERICA, LLC (XC36KHSA2U)'
and entitlements from provisioning profile: AppStore__North_America.mobileprovision
Repackaging as blubNorthAmericaWithSwift.ipa
Process complete
```

**Comment by mfurtak** [comment]
Hi @iphil - thanks for the report! :+1: 

Can you give us the exact command you are running, and any configuration information you're using in your project that would apply? Thanks!

**Comment by lacostej** [comment]
@iphil the logging is now only enabled in verbose mode, use `-v` to get it if you run from the CLI.
Change happened here 00a256ef

**Comment by lacostej** [comment]
And if you are using the `sigh` command, the attached PR will allow you to add the `-v` option of the resign script by using `sigh resign --verbose`.

@mfurtak please review :)

**Comment by asfalcone** [comment]
Merged #4021! Thanks @lacostej :racehorse:

**Comment by iphil** [comment]
Thank you @asfalcone!
